.github/workflows/test.yml

```diff
@@ -36,11 +36,16 @@ jobs:
         uses: actions/cache@v2
         with:
           path: '${{ env.ELM_HOME }}'
-          key: elm-${{ matrix.os }}-${{ hashFiles('elm/**/elm.json', 'example-*/**/elm.json', 'tests/**/elm.json') }}
+          key: elm-${{ matrix.os }}-${{ hashFiles('elm-tooling.json', 'elm/**/elm.json', 'example-*/**/elm.json', 'tests/**/elm.json') }}
 
       - name: npm ci
         if: steps.cache-node_modules.outputs.cache-hit != 'true'
         run: npm ci
+        env:
+          NO_ELM_TOOLING_INSTALL: 1
+
+      - name: elm-tooling install
+        run: npx --no-install elm-tooling install
```

Comment:
Yes! I’ve tried to explain the CI setup here: https://github.com/lydell/elm-tooling.json/#ci

Reply:
What would happen if we combined these steps?

Reply:
Then I guess we would have to make the `node_modules` cache depend on `elm-tooling.json` (even though it doesn’t, really), so that the `npm ci` step runs if `elm-tooling.json` changes. This means that we unnecessarily re-install `node_modules` when we only needed to re-install elm tools.

Reply:
Ah I see 👍